wallet/src/Pos/Wallet/Redirect.hs

```diff
@@ -78,7 +79,11 @@ networkChainDifficultyWebWallet = getLastKnownHeader >>= \case
 localChainDifficultyWebWallet
     :: forall ssc ctx m. BlockchainInfoEnv ssc ctx m
     => m ChainDifficulty
-localChainDifficultyWebWallet = view difficultyL <$> getTipHeader @ssc
+localChainDifficultyWebWallet = do
+  -- Workaround: Make local chain difficulty monotonic
+  prevMaxDifficulty <- GS.getMaxSeenDifficulty
```

Comment:
It's safer to use `GS.getMaxSeenDifficultyMaybe` here, though it's initialized on start and database without `maxSeenDifficulty` is basically invalid. So I don't really mind.

Reply:
Done!